Task_objects/index.js

```diff
@@ -1,24 +1,42 @@
-const cat = {
-  name: "Bertie",
-  breed: "Cymric",
-  color: "white",
-  greeting: function () {
-    console.log("Meow!");
-  },
-};
-
-// Write the code for 1st task below
-
-
```

Comment:
The cat.greeting statement on line 12 is missing parentheses to execute the function. It should be cat.greeting().